config/application.rb

```diff
@@ -66,7 +66,7 @@ class Application < Rails::Application
     end
 
     config.active_job.queue_adapter = :shoryuken
-    config.active_job.queue_name_prefix = Rails.env
+    config.active_job.queue_name_prefix = ENV['ANNOUNCE_RESOURCE_PREFIX'] || ENV['RAILS_ENV']
```

Comment:
I couldn't remember if we were setting this explicitly, this looks right, and I like having the fallback to env, esp for local dev